[CSPM RN] 23.7.1 - June 30 publish

Aishbn opened the pull request.

Aishbn committed:
RLP-104280 - Includes Google Cloud Translation

Aishbn committed:
RLP-105626 - Includes OJK under AWS

Aishbn committed:
Remove Fixed Issue RLP-103064

Aishbn committed:
Includes July 2023 in the list

Aishbn committed:
RLP-104171 - AG update to include Singapore

Aishbn committed:
Initial draft July 2023

Aishbn committed:
Includes LA changes before review 7.1

Aishbn committed:
RLP-106201 - New regions added for AWS.

Aishbn committed:
RLP-103734 - Added blurb for integrated view

Aishbn committed:
New lga image for RLP-103734

Aishbn committed:
LGA Image name update

Aishbn committed:
iam-policy-updates

Aishbn committed:
Post review updates v1

Pubs-MV committed:
added RLP-106498 for credit/licensing requirements

Aishbn committed:
Removed OCI ingestions - moved to 8.1

Aishbn committed:
KN -Removed RLP-30805, RLP-27773, and UAI issue

Aishbn committed:
review comment updates v2

Aishbn committed:
Minor AG Fixes from Ashwini.

anaghapamidi committed:
[CodeSec RN] Add 23.7.1 release notes

anaghapamidi commented:
Summary of changes for[ this commit](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1312/commits/2a11430cd530cabd78d6aed3e2aa101dfd23433e):

Modified files:

- cspm/rn/book.yml
-  cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-2023.adoc

New files:
- cspm/rn/_graphics/codesec-rn-23.7.1.png
- cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-july-2023.adoc

anaghapamidi committed:
[CodeSec RN] Add LA for 7.1

anaghapamidi commented:
Summary of changes for[ this commit](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1312/commits/1c9b9956de52e411c3f67a81bdcea1af5cd02765):

Modified files:
- cspm/rn/prisma-cloud-code-security-release-information/look-ahead-planned-updates-prisma-cloud-code-security.ado

anaghapamidi committed:
[CodeSec RN] Add G's feedback

Pubs-MV committed:
Update features-introduced-in-july-2023.adoc

@jenjoe22, some edits to make the featur name a noun phrase and to remove the reference to a  "feature"
